parser.go

```diff
@@ -43,53 +43,63 @@ func parsePackage(ctx *parseContext) {
 
 	_, ok := ctx.Env.PackageByName(ctx.File.Name.Name)
 	if !ok {
-		var comment []string
+		var comments []string
 		if ctx.File.Doc != nil {
-			comment = parseComments(ctx.File.Doc)
+			for _, t := range ctx.File.Comments {
+				parseComments(t, &comments)
+			}
 		}
 		pkg := &Package{
 			Name:    ctx.File.Name.Name,
-			Comment: comment,
+			Comment: comments,
 		}
 		ctx.Env.AppendPackage(pkg)
 		ctx.PackagesMap[ctx.File.Name.Name] = pkg
 	}
 }
 
-func parseComments(doc *ast.CommentGroup) []string {
+func parseComments(doc *ast.CommentGroup, c *[]string) {
```

Comment:
Why did you prefer to receive it as a pointer?

Reply:
I don't know, but i will go see if this is necessary.

Reply:
This has been resolved in branch **builtin-parse**. No is need use pointers.